src/printer.js

```diff
@@ -1,5 +1,14 @@
 "use strict";
 
+// other cases that are unstable/problematic:
```

Comment:
(for posterity, vjeux filed as https://github.com/babel/babylon/issues/457 and https://github.com/babel/babylon/issues/456, as well as several comment-related issues on this repo)